scripts/vs-code-debug-script.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
-export VSC_DEBUG='python -m debugpy --listen 0.0.0.0:5678 -m flask run --no-debugger --no-reload -p 4001 -h 0.0.0.0'
+export VSC_DEBUG='python -m debugpy --listen 0.0.0.0:5678 -m flask run --no-debugger --no-reload -p 4001 -h 0.0.0.0 --cert=/app-certs/cert.pem --key=/app-certs/cert-key.pem'
```

Comment:
**issue:** This should be `/app-certs/key.pem` not `/app-certs/cert-key.pem`

Reply:
Good spot

Reply:
Edited the above comment as my suggestion didn't resolve the issue, so I've just put the error rather than the potential fix

Reply:
Have pushed a fix - seems to work for me now - can you re-test?

Reply:
Testing now

Reply:
Have you generated the certs?

Reply:
Trying to debug currently

Reply:
I think I need to reword the readme. You need the certs all of the time, not just when you have the account. Lemme tweak that...

Reply:
@wjrm500 you might need to `sudo chmod 666 certs/*`

Reply:
mkcert should set the right access bits.

If you've had to use the superuser account you may need to chown the files rather than chmod them.

Happy to chat and debug when I have internet back on my laptop later...

Reply:
mkcert should set the right access bits.

If you've had to use the superuser account you may need to chown the files rather than chmod them.

Happy to chat and debug when I have internet back on my laptop later...

Reply:
Can confirm after a huddle with Sam that everything is now working for me locally, including the debug script. Will approve but suggest updating the `README.md` a bit to help Linux non-natives like me 🥲